scripts/create-local-tag-for-release

```diff
@@ -94,16 +99,23 @@ sync_local_tags_from_remote() {
     # fetch remote tags
     git fetch the-real-upstream --tags
 
+    # record the latest release tag in remote, before creating a new tag
+    CURR_REMOTE_RELEASE_TAG=$(get_latest_tag)
```

Comment:
is there validation on the user's -v input?  For example, if I put `-v 1.5.0` will this CLI reject it or add the prepended `v` ?  I didn't see anything like that, but I  may have missed it.

Reply:
Yep.
See `validate_args()` function.

```
create-local-tag-for-release -v 1.1.1
❌ Invalid new tag specified 1.1.1. Examples: v1.2.3, v1.2.3-dirty
```